plugins/generator-1.19.4/forge-1.19.4/procedures/team_add_entity.java.ftl

```diff
@@ -0,0 +1,6 @@
+{
+	Level _level = entity.level();

```

Comment:
I meant to not use dependencies at all, since you already get ${input$entity}.

We no longer require entity as dependency but as input in all procedure blocks.

Reply:
Oh ok, sorry. I didn't understand correctly